cpp/ycm/ClangCompleter/Diagnostic.h

```diff
@@ -53,6 +81,8 @@ struct Diagnostic {
   std::string text_;
 
   std::string long_formatted_text_;
+
+  std::vector< FixItChunk > fixits_;
```

Comment:
I don't feel strongly either way, I just wanted to understand. Code comment clarifying this would ensure others don't get confused.